gapipy/models/base.py

```diff
@@ -126,7 +126,7 @@ def _set_resource_field(self, field, value):
             for v in value:
                 stub.append(self._model_cls(field)(v, stub=True))
         elif value:
-            stub = self._model_cls(field)(value, stub=True)
+            stub = self._model_cls(field)(value, self._client, stub=True)
         setattr(self, field, stub)
 
     def _set_resource_collection_field(self, field, value):
```

Comment:
It's in this method that we use the `self._client`. It's the _only_ instance of using that property of the `BaseModel` class. There must be a Better Way™